dev-lang/luajit/luajit-2.1.0_beta2.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A Just-In-Time Compiler (JIT) for the Lua programming language"
+DESCRIPTION="Lua is a powerful, dynamic and light-weight programming \
+language. It may be embedded or used as a general-purpose, stand-alone \
+language. LuaJIT is a Just-In-Time Compiler (JIT) for it."
+HOMEPAGE="http://luajit.org/luajit.html"
+COPYRIGHT="2005-2015 Mike Pall"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="http://luajit.org/download/LuaJIT-$portVersion.tar.gz"
+CHECKSUM_SHA256="713924ca034b9d99c84a0b7b701794c359ffb54f8e3aa2b84fad52d98384cf47"
+SOURCE_DIR="LuaJIT-2.1.0-beta2"
+PATCHES="luajit-$portVersion.patchset"
+
+ARCHITECTURES="x86 !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	luajit$secondaryArchSuffix = $portVersion compat >= 5.1
+	lib:libluajit_5.1$secondaryArchSuffix = $portVersion compat >= 2
+	cmd:luajit_$portVersion$secondaryArchSuffix
+	cmd:luajit$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	luajit${secondaryArchSuffix}_devel = $portVersion compat >= 5.1
+	devel:libluajit_5.1$secondaryArchSuffix = $portVersion compat >= 2
+	"
+REQUIRES_devel="
+	luajit$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	# Use amalg target as recommended on luajit homepage to get better performance
+	make $jobArgs amalg PREFIX=$prefix
+}
+
+INSTALL()
+{
+	make install PREFIX=$prefix INSTALL_LIB="$libDir" INSTALL_BIN="$binDir" \
+		INSTALL_INC="$includeDir" INSTALL_SHARE="$dataDir" \
+		INSTALL_MAN="$manDir/man1" INSTALL_LMOD="$dataDir/lua/5.1"
+
+	ln -s libluajit-5.1.so.2.1.0 $libDir/libluajit-5.1.so.2
```

Comment:
This doesn't look right. You're symlinking a file in the local directory to `$libDir`?